model.py

```diff
@@ -96,13 +103,23 @@ def sessionmaker(cls, url=None):
         return sessionmaker(bind=engine)
 
     @classmethod
-    def initialize(cls, url):
+    def initialize(cls, url: str, testing=False) -> Tuple[Engine, Connection]:
+        """Initialize the database connection
+        Create all the database tables from the models
+        Optionally, run the alembic migration scripts
+
+        :param db_url: The Database connection url
+        :param testing: Whether we are in a test environment or not"""
         if url in cls.engine_for_url:
             engine = cls.engine_for_url[url]
             return engine, engine.connect()
 
         engine = cls.engine(url)
 
+        # Run the migrations only if we're not TESTING
+        if not testing:
```

Comment:
Hmm.. this made sense when the `db_migrate` module was being invoked from the `app.py` file, and we needed to avoid import race conditions from the testing files.
Now, I think, we can remove this

Reply:
Ok, I remember why this exists, the `alembic/env.py` pulls the `SIMPLIFIED_PRODUCTION_DATABASE` env var through `Configuration.database_url()`.
In order for it to hit the right database we can easily just check `TESTING in os.environ` but that's more parts of the code where we're maintaining the same testing validations.
Since we never _really_ need to do a migration in the testing environment I don't see the need to add this extra bit of code.